composer.json

```diff
@@ -6,7 +6,8 @@
     "php-di/php-di": "*",
     "middlewares/trailing-slash": "*",
     "league/commonmark": "*",
-    "guzzlehttp/guzzle": "*"
+    "guzzlehttp/guzzle": "*",
+    "swagger-api/swagger-ui": "^5.18"
```

Comment:
i set all version to `*`, because then I thought is renovate able to update the dependencies. (but maybe I'm wrong), but at least streamlined.
or don't you use the newest version?

dependencies are pinned anyway in composer.lock